vnext/Microsoft.ReactNative/Microsoft.ReactNative.vcxproj

```diff
@@ -115,11 +117,12 @@
         $(ReactNativeWindowsDir)Shared;
         $(ReactNativeWindowsDir)Shared\tracing;
         $(ReactNativeWindowsDir)include\Shared;
+        $(ReactNativeDir)\ReactCommon\react\renderer\graphics\platform\cxx;
         $(YogaDir);
         $(GeneratedFilesDir);
         %(AdditionalIncludeDirectories)
       </AdditionalIncludeDirectories>
-      <PreprocessorDefinitions>FOLLY_NO_CONFIG;NOMINMAX;_HAS_AUTO_PTR_ETC;RN_EXPORT=;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <PreprocessorDefinitions>FOLLY_NO_CONFIG;NOMINMAX;_HAS_AUTO_PTR_ETC;M_PI=3.14159265358979323846264338327950288;RN_EXPORT=;%(PreprocessorDefinitions)</PreprocessorDefinitions>
```

Comment:
wat 😁

Reply:
Replaced with _USE_MATH_DEFINES.  -- so we use the "normal" definition of M_PI.